examples/predictors/lstm/xprize_predictor.py

```diff
@@ -212,7 +215,8 @@ def _load_original_data(self, data_url, cutoff_date=None):
         latest_df = pd.read_csv(data_url,
                                 parse_dates=['Date'],
                                 encoding="ISO-8859-1",
-                                error_bad_lines=False)
+                                error_bad_lines=False,
+                                low_memory=False)
```

Comment:
Silencing a Pandas warning

Reply:
I've fixed it in https://github.com/leaf-ai/covid-xprize/pull/93
The `RegionName` and `RegionCode` columns contain a string or NaN, preventing Pandas from figuring out the type of the columns. Fixing the type to `str` is enough.